llpc/patch/llpcPatchIntrinsicSimplify.cpp

```diff
@@ -368,6 +381,31 @@ Value* PatchIntrinsicSimplify::SimplifyTrigonometric(
     return pNewCall;
 }
 
+// =====================================================================================================================
+// Test if an intrinsic can be simplified.
```

Comment:
Test -> Checks
We often "Checks' in those judgement methods. This is not a issue, just a suggestion :)

Reply:
I have updated the comment for consistency.